content/Products/OpenshiftMonitoring/collecting_metrics.md

```diff
@@ -233,17 +233,16 @@ spec:
     port: metrics
     scheme: https
     tlsConfig:
```

Comment:
Is this field still needed? The spec marks it optional. If its needed for another reason maybe a comment is in order?

Reply:
usually you need to configure the `serverName`.

Reply:
:facepalm: of course :)